packages/syft/src/syft/core/tensor/smpc/party.py

```diff
@@ -38,10 +38,14 @@ def get_protobuf_schema() -> GeneratedProtocolMessageType:
         return Party_PB
 
     def __hash__(self) -> int:
+        # TODO: Rasswanth, George, Trask this takes into consideration a hashing based on url and port
+        # that we login only once
+        # Is that sufficient?
```

Comment:
I think it is sufficient as we send the context at MPCTensor level,but yeah we need to give it some thought as the register functionality might change.